blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md

```diff
@@ -0,0 +1,82 @@
+![image](/img/blog/2017/codeaholics_pairprogramming.jpg)
+<div class="image-credits"> Pair Programming at Open Labs Hackerspace. (Photo: Jona & Xheni, Creative Commons Attribution-Share Alike 4.0 International.)</div>
+
+
+### __Nextcloud contributions__
+
+There is an albanian idiom "-Fillimi i mbarë, gjysma e punës" (English equivalent: Well begun, is half done), therefore we created our first PR on our first week and we began contributing on Nextcloud by fixing some starter issues, opening new issues, testing and reviewing etc.
+
+Nowadays we are mainly focused on developing a new feature: "favorite contacts”, allowing users to quickly favourite some of their contacts.  We are confident to have this feature finished by the end of the summer.
+
+What will be next? We certainly will continue contributing on other Nextcloud apps, even after the summer ends because we really love the vibe that this community has and the friendly environment.
+
+
+### __Girl Power at the Nextcloud conference!__
+
+One of the best things about working on the open source projects are the conferences. [Codeaholics](https://twitter.com/codeaholics_AL) joined the [Nextcloud Conference](https://nextcloud.com/conf/) on 22-29 August in Berlin, Germany. We were the early birds who arrived in the morning on the first day, waiting for other contributors to join the conference.
+![image](/img/blog/2017/codeaholics_nextcloud_conference.jpg)
+<div class="image-credits">Team Codeaholics at Nextcloud Conference. (Photo: Christoph Wurst, Creative Commons Attribution-Share Alike 4.0 International.)</div>
+
+Finally, we had the chance to meet our coaches Morris and Joas in person, our mentor joined us as well. We had four amazing days full of hacking and working on specific tasks on Nextcloud. It's very helpful when you have your coaches nearby so you can directly ask them and clarify in case of any doubts. In addition to meeting our coaches we also met many other contributors during our stay in Berlin. Fun fact: We knew most of them already by their GitHub usernames, that's why the first question we asked them was their github username. ;)
+
+We attended some talks related to the collaboration between Red Hat and Nextcloud, Collabora Online, how the Technical University of Berlin uses Nextcloud, Scalability Recommendations, Security etc.
+
+On the weekend, there were some lightning talks and workshops where we also shared [our experience](https://www.youtube.com/watch?v=OpMtsE_MICs) as RGSoC students with them (a lot of applauses in the end, yaaay)! It was a wonderful conference and a great opportunity to network with the community behind Nextcloud, which makes you feel more motivated to keep contributing.
+![image](/img/blog/2017/codeaholics_nextcloudconference.jpg)
+<div class="image-credits">Nextcloud Conference. (Photo:Raghu Nayyar)</div>
+And just for the record, coding while drinking Club Mate is one of the best feelings ever. (Try it :P)
```

Comment:
It would be better to have one line space between image and normat text [not image-credits though]. Can you please do that over here?